pkg/kubelet/apis/cri/v1alpha1/runtime/api.pb.go

```diff
@@ -1350,6 +1330,12 @@ type LinuxContainerSecurityContext struct {
 	//    (localhost) by name. The possible profile names are detailed at
 	//    http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference
 	ApparmorProfile string `protobuf:"bytes,9,opt,name=apparmor_profile,json=apparmorProfile,proto3" json:"apparmor_profile,omitempty"`
+	// Seccomp profile for the container, candidate values are:
+	// * runtime/default: the default profile for the container runtime
```

Comment:
We may want to define these values in the API too, but since this field is just to help with the transition, this is ok.